sections/featured-collection.liquid

```diff
@@ -207,6 +213,12 @@
       "label": "t:sections.featured-collection.settings.show_rating.label",
       "info": "t:sections.featured-collection.settings.show_rating.info"
     },
+    {
+      "type": "checkbox",
+      "id": "enable_quick_buy",
+      "default": false,
+      "label": "t:sections.featured-collection.settings.enable_quick_buy.label"
+    },    
```

Comment:
I ran `theme-check` on your `quick-buy` branch and it found 2 `style` suggestions, which can be auto-corrected. If you run `shopify theme check` or `theme-check` (using the standalone version), you'll see what I'm referring to but I'll try to outline them here:

**Offense 1 of 2** - `featured-collection.liquid`

```ruby
sections/featured-collection.liquid:103: style: SchemaJsonFormat: JSON formatting could be improved.
	{% schema %}
```

When you use the auto-correct flag `shopify theme check -a`, you'll be able to see what it fixed. For the `featured-collection.liquid`, it will [remove the empty whitespace from line 221](https://user-images.githubusercontent.com/28404165/156794380-814af91e-7ab3-44c3-9a7c-520f4d2e2989.png)

**Context**: There are currently **22 errors/offenses**. 2 of these can be auto-corrected: 1 mentioned here (above) and I'll add another comment for the other auto-correctable one). The other 20 errors/offenses are being fixed in the **Theme check manual fixes PR**.